profiles/management/commands/retire_users.py

```diff
@@ -0,0 +1,93 @@
+"""
+Retire user from MM
+"""
+from argparse import RawTextHelpFormatter
+from django.contrib.auth.models import User
+from django.core.management import BaseCommand, CommandError
+from social_django.models import UserSocialAuth
+
+from dashboard.models import ProgramEnrollment
+
+
+class Command(BaseCommand):
+    """
+    Retire user from MicroMasters
+    """
+    help = """
+Retire one or multiple users. For single user use:\n
+`./manage.py retire_users --user=foo` or do \n
+`./manage.py retire_users -u foo` \n
+
+For multiple users, add arg `--user` for each user i.e:\n
+`./manage.py retire_users --user=foo --user=bar --user=baz` or do \n
+`./manage.py retire_users -u foo -u bar -u baz`
+"""
+
+    def create_parser(self, prog_name, subcommand):
+        """
+        create parser to add new line in help text.
+        """
+        parser = super(Command, self).create_parser(prog_name, subcommand)
+        parser.formatter_class = RawTextHelpFormatter
+        return parser
+
+    def add_arguments(self, parser):
+        """create args"""
+        # pylint: disable=expression-not-assigned
+        parser.add_argument(
+            '-u',
+            '--user',
+            action='append',
+            default=[],
+            dest='users',
+            help="Single or multiple user name"
+        ),
+
+    def command_message(self, message, is_error=False):
+        """
+        display and log message
+        """
+        self.stdout.write(message, self.style.ERROR)
```

Comment:
This is going to print everything as an error, which they're not all, you're probably better off just inlining this and doing the correct calls as-needed